scripts/actions/send-and-update-translation-queue.js

```diff
@@ -28,19 +28,31 @@ const DOCS_SITE_URL = 'https://docs.newrelic.com';
  * @param {Object<string, string[]>} locales The queue of slugs to be translated.
  * @returns {Object<string, Promise<Page[]>>}
  */
-const getContent = (locales) =>
-  Object.entries(locales).reduce((acc, [locale, slugs]) => {
+const getContent = (locales) => {
+  return Object.entries(locales).reduce((acc, [locale, slugs]) => {
     return {
       ...acc,
       [locale]: Promise.all(
-        slugs.map(async (slug) => {
-          const mdx = fs.readFileSync(path.join(process.cwd(), slug));
-          const html = await serializeMDX(mdx);
-          return { file: slug, html };
-        })
+        slugs
+          .filter((slug) => {
+            /**
+             * if a doc doesn't exist, it must have been renamed or deleted. in
+             * that case, it is safe to ignore. if we skip including a doc in
+             * this step, it won't become a failed upload, and will then be
+             * cleaned up from the queue.
```

Comment:
Kinda nitpicky grammar thing, but if we're going to be using periods (`.`), then we should capitalize the first letter of each sentence.

Reply:
🙈

Reply:
It's a matter of preference. I personally like lots of comments in the code, but I know that it's not everyones cup of tea 🍵

Reply:
Tossing in my vote to keep the comment!